projects/keeperdao/index.js

```diff
@@ -111,26 +117,127 @@ async function getMarkets(block) {
   }
 }
 
-async function tvl(timestamp, block) {
-  let balances = {};
-  let markets = await getMarkets(block);
+// Calculates all the token balances in Hiding Vault NFTs minted till the given block
+async function getHidingVaultBalances(timestamp, block) {
+  let hidingVaultBalances = {}
 
-  // Get V1 tokens locked
-  let v1Locked = await sdk.api.abi.multiCall({
-    block,
-    calls: _.map(markets, (data, token) => ({
-      target: token,
-      params: liquidityPool,
-    })),
-    abi: 'erc20:balanceOf',
-  });
+  // Track TVL of Hiding Vaults after it went live
+  if(block > HIDING_VAULT_START_BLOCK_NUMBER) {
+    // Get total Hiding Vault NFT count by calling totalSupply on Hiding Vault Contract
+    let noOfHidingVaults = (await sdk.api.erc20.totalSupply({
+      target: HIDING_VAULT_CONTRACT,
+      block: block 
+    })).output;
+    
+    const indexRange = _.range(1, parseInt(noOfHidingVaults) + 1);
+
+    // Query Hiding Vault Contract's 'tokenByIndex' with index to get individual HidingVaultNFTs
```

Comment:
tokenByIndex is also part of the ERC721 enumerable spec, not sure if there's already a library for this or not